aviary/subsystems/propulsion/test/test_prop_performance.py

```diff
@@ -103,6 +103,10 @@ def test_case_0_1_2(self):
         prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED,
                      [800.0, 800.0, 750.0], units="ft/s")
         prob.set_val(Dynamic.Mission.SHAFT_POWER, [1850.0, 1850.0, 900.0], units="hp")
+        prob.set_val(Dynamic.Mission.PERCENT_ROTOR_RPM_CORRECTED,
+                     [1.0], units="unitless")
```

Comment:
here and below, should this be `[1,1,1]`?

Reply:
I guess we can assume that this parameter varies but in GASP, it is a constant in each phase (takeoff, climb, cruise). I changed it to [1,1,1].

Reply:
thanks